dashboard/test/ui/runner.rb

```diff
@@ -343,6 +343,10 @@ def run_tests(env, feature, target_platform, arguments, log_prefix)
                         feature_name: feature.include?(".feature") ? feature.split(".feature")[0] : feature,
                         target_browser: target_platform}
     # Metrics for individual feature runs. They will be flushed once all of them run
+    unless eyes_succeeded
+      Infrastructure::Logger.put("runner_feature_eyes_diff", 1, extra_dimensions)
+    end
+
```

Comment:
I'd suggest either making this a one-liner like so:

```suggestion
      Infrastructure::Logger.put("runner_feature_eyes_diff", 1, extra_dimensions) unless eyes_succeeded
```

Or maintaining parallelism with the other logs by conditionally logging 0 or 1 like so:

```suggestion
      Infrastructure::Logger.put("runner_feature_eyes_diff", eyes_succeeded ? 1 : 0, extra_dimensions)
```

Reply:
I definitely agree we should be logging 0's

Reply:
Making the one liner sounds good to me. 

Can you pl help me see the value of logging 0's though. In case of no eyes diffs, those are reported as `runner_feature_success` event. Also, I'm curious how the 0's help in general, is there a cloudwatch dashboard method that can help with computing % by emitting both 0's and 1's for complementary events?

Reply:
You're logging 0's for `runner_feature_failure` and `runner_feature_success`, so for consistency, shouldn't you do the same here? (or make those also follow the `log 1 unless condition` pattern?

Reply:
This is the inconsistency,

```
Infrastructure::Logger.put("runner_feature_eyes_diff", 1, extra_dimensions) unless eyes_succeeded
Infrastructure::Logger.put("runner_feature_success", cucumber_succeeded ? 1 : 0, extra_dimensions)
Infrastructure::Logger.put("runner_feature_failure", cucumber_succeeded ? 0 : 1, extra_dimensions)
```
 
Should you do:

```
Infrastructure::Logger.put("runner_feature_eyes_diff", 1, extra_dimensions) unless eyes_succeeded
Infrastructure::Logger.put("runner_feature_success", 1, extra_dimensions) if cucumber_succeeded
Infrastructure::Logger.put("runner_feature_failure", 1, extra_dimensions) unless cucumber_succeeded
```

Reply:
The metrics for `runner_feature_success` and `runner_feature_failure` existed earlier, so I am not sure what the intention was behind following the 0/1 approach. From my research on the various methods that cloud watch provides to plot these metrics, I don't see a way to leverage these duplicate metrics.

So, I'm inclined to change those also to avoid publishing duplicate metrics, that might seem reasonable give we are seeing  errors in widget saying the total number of time series is over the limit. Sending another iteration with that update.

Reply:
Yeah, I think we'll get more value from just sending 1's when the event happens, and not sending otherwise.